src/node_url.cc

```diff
@@ -544,19 +544,6 @@ std::optional<std::string> FileURLToPath(Environment* env,
 #endif  // _WIN32
 }
 
-// Reverse the logic applied by path.toNamespacedPath() to create a
-// namespace-prefixed path.
-void FromNamespacedPath(std::string* path) {
-#ifdef _WIN32
-  if (path->compare(0, 8, "\\\\?\\UNC\\", 8) == 0) {
```

Comment:
Why not keep support to v18 and v20?

Reply:
Because it won't make a difference. This function is unlikely going to change in the future.